errai-ioc/src/main/java/org/jboss/errai/ioc/rebind/ioc/bootstrapper/IOCProcessor.java

```diff
@@ -1296,12 +1295,10 @@ protected boolean getPropertyValue(final String propName,
       return erraiConfiguration.app().asyncBeanManager();
     }
 
-    final String propertyValue = EnvUtil.getEnvironmentConfig().getFrameworkOrSystemProperty(propName);
-    if (propertyValue == null) {
-      return matchByDefault;
-    } else {
-      return caseSensitive ? propertyValue.equals(matchValue) : propertyValue.equalsIgnoreCase(matchValue);
-    }
+    return erraiConfiguration.app()
+            .custom(propName)
```

Comment:
Okay, I see now why you need the custom property for IOC. To handle `@EnabledByProperty`, right?

Reply:
Exactly :)